ComfortZoneApp/app/src/main/java/com/example/comfortzone/flight/ui/FlightFragment.java

```diff
@@ -214,10 +217,12 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
             @Override
             public void afterTextChanged(Editable s) {
-                String city = Normalizer.normalize(s, Normalizer.Form.NFD);
-                city = city.replaceAll("[^\\p{ASCII}]", "");
-                List<WeatherData> searchedCity = FilteringUtils.searchCity(city, cityList);
-                updateViewsList(searchedCity);
+                if (!s.toString().equals("")) {
```

Comment:
instead of comparing with empty string, you can just use `isEmpty()`: https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#isEmpty()